features/issue/api/editIssue/editIssue.ts

```diff
@@ -0,0 +1,24 @@
+import { EditIssueSchema } from '../../schemas'
+
```

Comment:
editIssue i useEditIssue nie ma potrzeby na dwa pliki rozbijać, może być w jednym, ale jak tam wolisz 
(To samo tyczy się delete, create itp)